api.md

```diff
@@ -852,8 +853,6 @@ are meant to implement data retention or perform tasks that will improve query
 performance on older chunks. Each policy is assigned a scheduled job
 which will be run in the background to enforce it.
 
-
-
 ## add_drop_chunks_policy() :enterprise_function: [](add_drop_chunks_policy)
```

Comment:
Is there a reason you didn't put a divider here?

Reply:
Was the first function under a section head, the other two section headers don't have the first function with a divider either...  (Happy to do that instead)